src/UI/examples/Popover/show_popover_with_vertical_scrollbars.php

```diff
@@ -0,0 +1,32 @@
+<?php
+function show_popover_with_vertical_scrollbars()
+{
+	global $DIC;
+	$factory = $DIC->ui()->factory();
+	$renderer = $DIC->ui()->renderer();
+
+	$series = [
+		'Breaking Bad',
+		'Big Bang Theory',
+		'Dexter',
+		'Better Call Saul',
+		'Narcos',
+		'Ray Donovan',
+		'Simpsons',
+		'South Park',
+		'Fargo',
+		'Bloodline',
+		'The Walking Dead',
+		'New Girl',
+		'Sons of Anarchy',
+		'How I Met Your Mother',
+	];
+	$list = $renderer->render($factory->listing()->unordered($series));
+	$content = "<div style='max-height: 200px; overflow-y: auto; padding-right: 10px;'>{$list}</div>";
```

Comment:
Of course not. This is just an example showing that it is possible to have vertical scrollbars. The `Popover` does not have a fixed `max-height`, since this may depend on the content component being rendered. Also we don't want to expose a `max-height` setting to the consumer, this should be the responsibility of the renderer.

What is the idea behind this? If the `Popover` is acting as container component, the renderer of the content component would be aware of the context, e.g. `Popover`. This is not currently implemented but I think there are plans to introduce this sort of context for renderers. 
As an example: The renderer of the chat would know wether it renders the chat in a sidebar or popover. If rendered in a `Popover`, the height would be restricted and, if necessary, vertical scrollbars are displayed.